L/libsigcpp/libsigcpp@2.12.0/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libsigcpp"
+version = v"2.12.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/libsigc++/2.12/libsigc%2B%2B-$(version).tar.xz",
+                  "1c466d2e64b34f9b118976eb21b138c37ed124d0f61497df2a90ce6c3d9fa3b5")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libsigc++*/
+mkdir meson
+cd meson
+meson --cross-file=${MESON_TARGET_TOOLCHAIN%.*}_gcc.meson
+ninja -j${nproc}
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
```

Comment:
Argh, I just realised that this should have expanded C++ string ABI also for BSD systems.  Why do you need GCC?

Reply:
Build fails with `clang++`, it looks like they've only taken fixes for it on the newer libsigc++-3.0 branches (and the fix does not directly backport)